beans/src/main/resources/org/ldaptive/beans/spring/spring-ext.xsd

```diff
@@ -109,6 +109,7 @@
       <xsd:extension base="baseAuthenticatorType">
         <xsd:attribute name="baseDn" type="xsd:string" use="required"/>
         <xsd:attribute name="userFilter" type="xsd:string" use="required"/>
+        <xsd:attribute name="passwordAttribute" type="xsd:string" use="optional"/>
```

Comment:
I didnt even know, if the change in XSD is needed. There is even no place where you specify how to change the password scheme from SHA to something different, so it was hard to guess, whats to correct place to add this this feature.